newsfragments/3227.feature

```diff
@@ -0,0 +1 @@
+Remove usage of child_ feature of Nevow (py3 port)
```

Comment:
Is it interesting to end users?  When Nevow is completely dropped as a dependency, that will probably be interesting.

Or maybe there should be a discussion about what the goals of the news file/changelog are - I guess Tahoe-LAFS has historically announced a lot more than I am interested in seeing in a news file, maybe I just need to get used to that (sounds like a discussion either way I guess).